huaweicloud/services/lts/resource_huaweicloud_lts_cce_access.go

```diff
@@ -0,0 +1,530 @@
+	createCceAccessConfigResp, err := ltsClient.Request("POST", createCceAccessConfigPath, &createCceAccessConfigOpt)
+	if err != nil {
+		return diag.Errorf("error creating CCE access config: %s", err)
+	}
+
+	createCceAccessConfigRespBody, err := utils.FlattenResponse(createCceAccessConfigResp)
+	if err != nil {
+		return diag.FromErr(err)
+	}
+
+	id, err := jmespath.Search("access_config_id", createCceAccessConfigRespBody)
+	if err != nil {
+		return diag.Errorf("error creating CCE access config: ID is not found in API response")
+	}
+	d.SetId(id.(string))
+
+	return resourceCceAccessConfigRead(ctx, d, meta)
+}
+
+func buildCreateCceAccessConfigBodyParams(d *schema.ResourceData) map[string]interface{} {
+	logInfoOpts := map[string]interface{}{
+		"log_group_id":  d.Get("log_group_id"),
+		"log_stream_id": d.Get("log_stream_id"),
+	}
+
+	bodyParams := map[string]interface{}{
+		"access_config_type":   "K8S_CCE",
+		"access_config_name":   d.Get("name"),
+		"access_config_detail": buildCceAccessConfigDeatilRequestBody(d.Get("access_config")),
+		"log_info":             logInfoOpts,
+		"host_group_info":      buildHostGroupInfoRequestBody(d),
+		"access_config_tag":    utils.ExpandResourceTags(d.Get("tags").(map[string]interface{})),
+		"cluster_id":           d.Get("cluster_id"),
+		"binary_collect":       utils.ValueIngoreEmpty(d.Get("binary_collect")),
+		"log_split":            utils.ValueIngoreEmpty(d.Get("log_split")),
+	}
+	return bodyParams
+}
+
+func buildCceAccessConfigDeatilRequestBody(rawParams interface{}) map[string]interface{} {
+	if rawArray, ok := rawParams.([]interface{}); ok {
+		if len(rawArray) == 0 {
+			return nil
+		}
+		raw, ok := rawArray[0].(map[string]interface{})
+		if !ok {
+			return nil
+		}
+
+		params := map[string]interface{}{
+			"pathType":           raw["path_type"],
+			"paths":              utils.ValueIngoreEmpty(raw["paths"]),
+			"black_paths":        utils.ValueIngoreEmpty(raw["black_paths"]),
+			"format":             buildHostAccessConfigFormatRequestBody(raw),
+			"windows_log_info":   buildHostAccessConfigWindowsLogInfoRequestBody(raw["windows_log_info"]),
+			"stdout":             utils.ValueIngoreEmpty(raw["stdout"]),
+			"stderr":             utils.ValueIngoreEmpty(raw["stderr"]),
+			"namespaceRegex":     utils.ValueIngoreEmpty(raw["name_space_regex"]),
+			"podNameRegex":       utils.ValueIngoreEmpty(raw["pod_name_regex"]),
+			"containerNameRegex": utils.ValueIngoreEmpty(raw["container_name_regex"]),
+			"logLabels":          raw["log_labels"],
+			"includeLabels":      raw["include_labels"],
+			"excludeLabels":      raw["exclude_labels"],
+			"logEnvs":            raw["log_envs"],
+			"includeEnvs":        raw["include_envs"],
+			"excludeEnvs":        raw["exclude_envs"],
+			"logK8s":             raw["log_k8s"],
+			"includeK8sLabels":   raw["include_k8s_labels"],
+			"excludeK8sLabels":   raw["exclude_k8s_labels"],
+		}
+		return params
+	}
+	return nil
```

Comment:
if the type of access configuration parameter is not interface{} array, just return nil? shall we record this error in the log?
I suggest you to check not ok first and record error, it is more friendly for other developer.
```
	rawArray, ok := rawParams.([]interface{})
	if !ok {
		log.Printf("...")
	}
	...
```

Reply:
done